lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1206,6 +1206,31 @@ class DecomposeAtenSoftplusOp : public OpRewritePattern<AtenSoftplusOp> {
 };
 } // namespace
 
+// Decompose aten.std.dim to sqrt(var.dim(x))
+namespace {
+class DecomposeAtenStdDimOp : public OpRewritePattern<AtenStdDimOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenStdDimOp op,
+		                PatternRewriter &rewriter) const override {
+    Value self = op.self();
+    BaseTensorType inputTensorType = self.getType().cast<BaseTensorType>();
+    if (!inputTensorType.hasDtype() ||
+        !inputTensorType.getDtype().isa<mlir::FloatType>()) {
+      return rewriter.notifyMatchFailure(op,
+                                         "aten.std.dim only supports floating type");
```

Comment:
nitpicky comment: I understand it is this way in many other places in the file, but I think a better error message here is something like:
aten.std.dim expects inputTensorType to be an mlir::FloatType. 

or at the very least change to "floating-point type"

Reply:
Changed the error message to `aten.std.dim expects input tensor of floating-point type`